include/gatekeeper_gk.h

```diff
@@ -19,17 +19,126 @@
 #ifndef _GATEKEEPER_GK_H_
 #define _GATEKEEPER_GK_H_
 
+#include <stdint.h>
+
 #include <rte_atomic.h>
 
+#define	START_PRIORITY		(38)
+/* Set @START_ALLOWANCE as the double size of a large DNS reply. */
+#define	START_ALLOWANCE		(8)
```

Comment:
I think both of these constants can go in `gk/main.c`, since they are only used there.